### DISCUSSION TITLE
Help me pls! Static Media Failure The following static media file failed to load: setmode.js

### DISCUSSION THREAD

**Discussion by Igor0790** [netbox-community/netbox#14258] [Help Wanted!]
![image](https://github.com/netbox-community/netbox/assets/53512805/6d8ddb68-2f1b-4059-9791-c944848b6d73)

How to fix this problem? This is a new netbox that I'm picking up for work needs. I followed all the recommendations. Did not help! At the same time, running locally, on the localhost, everything works. This problem only occurs when I turn on the netbox server on a specific IP.

**Comment by Igor0790** [comment]
Addition - Windows 10 OS, the database and redis are also installed locally.

**Reply by Igor0790** [reply]
I dont use of servers Apache or ngnix.

**Reply by candlerb** [reply]
You are not installing Netbox in the recommended way. In production, Netbox needs a reverse proxy which serves the static files: you will find this in [step 5](https://docs.netbox.dev/en/stable/installation/5-http-server/) of the installation instructions.

You cannot point clients directly at gunicorn.  It won't work because it doesn't serve the static files, as your error message shows.

You can instead run a [development server](https://docs.netbox.dev/en/stable/installation/3-netbox/#test-the-application) which does serve the static files from python, but this is not recommended for production use for multiple reasons.

**Reply by Igor0790** [reply]
Thank you very much for the recommendation. I'll take note and fix it.

**Comment by Chibraax** [comment]
I'm a bit late to the party, but if you are like me and just want to test locally (without a real server like nginx/apache) netbox and you got this error. 

Just set `DEBUG = True` in **_configuration.py_**.
When DEBUG is set to False, django doesnt serve static files

Remember, never set DEBUG to true in production environment.
